README.md

```diff
@@ -77,27 +76,24 @@ npm run create-md -- <filename> "<relative-path>"
 npm run create-md -- textarea "native/inputs"
 ```
 
-This command will create:
+```bash
+npm run create-md -- "links-buttons" "how-to-test/components"
```

Comment:
for this one -- it is generating a page like the web-checklist and native-checklist. It doesn't look much like other how to test pages. Maybe we can discuss this one